tests/roles/backend_services/templates/container_overrides.j2

```diff
@@ -45,6 +45,8 @@ spec:
     templates:
       openstack:
         containerImage: {{ container_registry }}/{{ container_namespace }}/openstack-mariadb:{{ container_tag }}
+      openstack-cell1:
```

Comment:
this shoudl be creating the mariadb-openstack-cell1 pod

Reply:
we do not used this in the ci job. i have upstaed the template we use in the latest revision.